reference/5.1/Microsoft.PowerShell.Core/Invoke-History.md

```diff
@@ -21,62 +21,76 @@ Invoke-History [[-Id] <String>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Invoke-History** cmdlet runs commands from the session history.
-You can pass objects representing the commands from Get-History to **Invoke-History**, or you can identify commands in the current history by using their ID number.
-To find the identification number of a command, use the **Get-History** cmdlet.
+
+The `Invoke-History` cmdlet runs commands from the session history. You can pass objects
+representing the commands from Get-History to `Invoke-History`, or you can identify commands in the
+current history by using their **Id**number. To find the identification number of a command, use the
```

Comment:
space between `**Id**number`.  It's not rendering right.

Reply:
Fixed